device/cig/x86_64-cig_cs6436_56p-r0/Cig-CS6436-56P/buffers_defaults_t1.j2

```diff
@@ -0,0 +1,72 @@
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '20971328' %}
+{% set ingress_lossy_pool_size =  '20971328' %}
+{% set egress_lossless_pool_size =  '20971328' %}
+{% set egress_lossy_pool_size =  '20971328' %}
+
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0, 48) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx)) %}{%- endif %}
+    {%- endfor %}
+    {%- for port_idx in range(48, 54) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (48 + (port_idx-48) * 4)) %}{%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "{{ ingress_lossless_pool_size }}",
+            "type": "ingress",
+            "mode": "static"
+        },
+        "ingress_lossy_pool": {
+            "size": "{{ ingress_lossy_pool_size }}",
+            "type": "ingress",
+            "mode": "dynamic"
+        },
+        "egress_lossless_pool": {
+            "size": "{{ egress_lossless_pool_size }}",
+            "type": "egress",
+            "mode": "static"
+        },
+        "egress_lossy_pool": {
+            "size": "{{ egress_lossy_pool_size }}",
+            "type": "egress",
+            "mode": "dynamic"
+        }
+    },
+    "BUFFER_PROFILE": {
+        "ingress_lossless_profile": {
+            "pool":"[BUFFER_POOL|ingress_lossless_pool]",
+            "xon":"78400",
+            "xoff":"132160",
```

Comment:
Why hard coded? not using the pg_profile_lookup.ini infrastructure?

Reply:
The default 'ingress_lossless_profile' is a configuration example for end user, and not bind and work on ports on these devices. 
pg_profle_lookup.ini used with buffermgrd, it will work whenever user configured ports' speed and cable length.